botbowl/core/pathfinding/python_pathfinding.py

```diff
@@ -557,7 +557,8 @@ def get_safest_path(game, player, position, from_position=None, allow_team_rerol
     """
     if from_position is not None and num_moves_used != 0:
         orig_player, orig_ball = _alter_state(game, player, from_position, num_moves_used)
-    can_handoff = game.is_handoff_available() and game.get_ball_carrier() == player
+    can_handoff = (game.is_handoff_available() or game.get_player_action_type() is PlayerActionType.HANDOFF) and \
+                  game.get_ball_carrier() == player
```

Comment:
I think this makes sense. My only suggestion is to make this expression a function with an understandable name. 

![image](https://user-images.githubusercontent.com/1433421/171265354-45f1ee63-14ca-4914-a47c-a476147d90c3.png)

Reply:
You know how bad I am at coming up with understandable names :laughing:  What should it be?